### PR TITLE
Change documentation details (closes #5525)

### LINKED ISSUE #5525: Module docs: 2 suggestions...
(1) The title of each module in the docs should be written "module-name Module" not "Module module-name" since this puts the important information first.

(2) The Imports section's structure should be changed. For example if this were the threadpool module it should be:

# Imports
`import threadpool`
Dependencies:  `cpuinfo`, `cpuload`, `locks`

With the dependencies links as now of course.

### PR DESCRIPTION
**Pull request by narimiran**
As proposed [here](https://github.com/nim-lang/Nim/issues/5525#issuecomment-286549519):

> (1) Or we remove the word "module" instead?
> (2) I think the "Imports" word should be changed to "Dependencies" instead.

### CODE REVIEW

**Comment by dom96** [comment]
While we're at it though, let's improve it even further, how about:

* <module_name> — Nim 0.19.0 documentation

This way it's much easier to see what version the documentation is for, especially for search engines.

File: compiler/docgen.nim [hunk 1]
```diff
@@ -873,7 +873,7 @@ proc generateTags*(d: PDoc, n: PNode, r: var Rope) =
 
 proc genSection(d: PDoc, kind: TSymKind) =
   const sectionNames: array[skTemp..skTemplate, string] = [
-    "Exports", "Imports", "Types", "Vars", "Lets", "Consts", "Vars", "Procs", "Funcs",
+    "Exports", "Dependencies", "Types", "Vars", "Lets", "Consts", "Vars", "Procs", "Funcs",
```
Comment (Araq) [inline compiler/docgen.nim]: Hum, I liked "imports" better...

File: nimdoc/testproject/expected/subdir/subdir_b/utils.html [hunk 2]
```diff
@@ -16,7 +16,7 @@
 <link href='https://fonts.googleapis.com/css?family=Source+Code+Pro:400,500,600' rel='stylesheet' type='text/css'/>
 
 <!-- CSS -->
-<title>Module utils</title>
+<title>utils — Nim v0.19.1</title>
```
Comment (Araq) [inline nimdoc/testproject/expected/subdir/subdir_b/utils.html]: Gah, so we need to touch this test then for every release? That sucks.

Comment (narimiran) [reply nimdoc/testproject/expected/subdir/subdir_b/utils.html]: Yeah, that's how I did it (manually changed it).

I can remove it so we have only `<title>utils</title>`, like it was in the first commit.

File: compiler/docgen.nim (hunk 1 above)
Comment (narimiran) [reply compiler/docgen.nim]: Eh, you were the one who proposed "Dependencies" :)

RFC? :rofl:

Comment (Araq) [reply compiler/docgen.nim]: I guess this means I changed my mind...

File: nimdoc/testproject/expected/subdir/subdir_b/utils.html (hunk 2 above)
Comment (Araq) [reply nimdoc/testproject/expected/subdir/subdir_b/utils.html]: So ... yeah, no version then or add some wildcard support for the nim doc tester.

Comment (narimiran) [reply nimdoc/testproject/expected/subdir/subdir_b/utils.html]: Ok, the new commit has no version, and uses "Imports" as before (making it a very small PR :)). Merge?

**Pull request merged**